docs/examples/aws_tags.rb

```diff
@@ -0,0 +1,23 @@
+require 'chef/provisioning/aws_driver'
+
+with_driver 'aws::us-west-2'
+
+machine 'ref-machine-1' do
+  action :allocate
+end
+
+aws_instance 'ref-machine-1' do
```

Comment:
I'd also like to add that even though tagging machines and load balancers isn't ideal with this branch, I would press that we still merge these changes pending a good review.  This is a feature that is much needed and we can improve it in a new branch.